### PR TITLE
Add detailed descriptions for some error codes to ease debug.

### PR DESCRIPTION
**Pull request by tdejoigny-ledger**

### CODE REVIEW

**Review by xchapron-ledger — COMMENTED** [review]
Would be nice to have a function mapping the error code to the message, so that it's not in 3 different places.

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ Use the following Target IDs (--targetId option) when running commands directly:
 
 | Device name      | Firmware Version                   | Target ID    |
 |------------------|------------------------------------|--------------|
+| `Flex`           | all                                | `0x33300004` |
+| `Stax`           | all                                | `0x33200004` |
 | `Nano S Plus`    | all                                | `0x33100004` |
 | `Nano X`         | < 2.2.1 (**developer units only**) | `0x33000004` |
 | `Nano X`         | \>=  2.2.1                         | `0x33000004` |
```
Comment (cedelavergne-ledger) [inline README.md:62]: Why keeping 2 different lines for NanoX, whereas the `Target ID` values are the same?

File: ledgerblue/comm.py [hunk 2]
```diff
@@ -209,6 +217,14 @@ def exchange(self, apdu, timeout=TIMEOUT):
...
 			if sw == 0x6e00:
 				possibleCause = "Unexpected state of device: verify that the right application is opened?"
+			if sw == 0x5515:
+				possibleCause = "Did you unlock the device?"
+			if sw == 0x6814:
+				possibleCause = "Unexpected target device: verify that your are using the right device?"
+			if sw == 0x511F:
+				possibleCause = "The OS version on your device does not seem compatible with the SDK version used to build the app"
+			if sw == 0x5120:
+				possibleCause = "Sideload is not supported on Nano X"
```
Comment (cedelavergne-ledger) [inline ledgerblue/comm.py:227]: Strange to duplicate the whole error codes check. Couldn't it be optimized in a common function?

File: README.md (hunk 1 above)
Comment (tdejoigny-ledger) [reply README.md]: true --> fixed

File: ledgerblue/comm.py (hunk 2 above)
Comment (tdejoigny-ledger) [reply ledgerblue/comm.py]: true --> I looked too quickly; I thought that some error codes differed depending on the communication method

**Comment by tdejoigny-ledger** [comment]
> Would be nice to have a function mapping the error code to the message, so that it's not in 3 different places.

yes agree

**Review by cedelavergne-ledger — APPROVED** [review]

**Review by tjoly-ledger — APPROVED** [review]
LGTM

**Pull request merged**